extensions/amp-next-page/1.0/page.js

```diff
@@ -20,12 +20,12 @@ import {devAssert} from '../../../src/log';
 
 /** @enum {number} */
 export const PageState = {
-  QUEUED: 0,
-  FETCHING: 1,
-  LOADED: 2,
-  FAILED: 3,
-  INSERTED: 4,
-  PAUSED: 5,
+  QUEUED: 1,
```

Comment:
Why are we shifting these?

Reply:
In general I like to start enums at 1 to avoid 0 being counted as null (should never happen but just to be safe)

Reply:
I feel like in most cases typecheck would guard against that, but not changing them is not an issue either way.